core/common/base.py

```diff
@@ -48,7 +49,8 @@ def send_data(self, data: Any, d_type: str = "str") -> None:
             self.socket.send(bytes(data, "utf-8"))
 
     def receive_data(self, size: int = 4096, data_type: str = str) -> Optional[Any]:
-        """
+        """Receive data to the connected socket.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)